azure-quantum/azure/quantum/qiskit/backends/ionq.py

```diff
@@ -53,7 +53,9 @@ def _prepare_job_metadata(self, circuit, **kwargs):
 
     def _translate_input(self, circuit, data_format, input_params, to_qir_kwargs={}):
         """ Translates the input values to the format expected by the AzureBackend. """
-        if data_format == "ionq.circuit.v1":
+        if "targetCapability" in input_params and input_params["targetCapability"] == "BasicExecution":
```

Comment:
So targetCapability can be "ionq" (line 34) or "BasicExecution"? Or these are different target capabilities?

Reply:
Yes that's right. I didn't know what to use for the default target capability for ionq, but I saw that quantuniuum has default capability of "openqasm", so I went with "ionq" for the default for ionq. It isn't a capability that would be recognized, I don't imagine, but the point is for there to be a default option other than "BasicExecution".

If there is a more correct way to set this up, please let me know, because I just took a guess here.

Reply:
I've removed the default targetCapability.